pom.xml

```diff
@@ -85,7 +85,7 @@
     <!-- properties for script build step that creates the config files for the artifacts -->
     <widgets.dir>widgets</widgets.dir>
     <docs.dir>docs</docs.dir>
-    <etl.versionRange>[4.0.0,10.0.0-SNAPSHOT)</etl.versionRange>
+    <app.parents>system:cdap-etl-batch[4.1.0,4.2.0-SNAPSHOT),system:cdap-etl-realtime[4.1.0,4.2.0-SNAPSHOT),system:cdap-data-pipeline[4.1.0,4.2.0-SNAPSHOT),system:cdap-data-streams[4.1.0,4.2.0-SNAPSHOT)</app.parents>
```

Comment:
Not entirely sure. It depend on it in older script version:
https://github.com/hydrator/wrangler/pull/45/files#diff-600376dffeb79835ede4a0b285078036L234

@albertshau Might be able to clarify if its needed or not.